internal/cmdparser/cmd.go

```diff
@@ -54,35 +71,77 @@ func (c *Cmd) AddFlag(options FlagOptions) {
 			panic("Only provide one type")
 		}
 		if options.Shorthand == "" {
-			c.command.PersistentFlags().IntVar(
-				options.Int,
-				options.Name,
-				options.DefaultInt,
-				options.Usage)
+			if options.Persistent {
```

Comment:
>.Persistent [](http://example.com/codeflow?start=13&length=11)

How about creating a wrapper over `c.command.[Persistent]Flags()` that implements the flags interface and handles the if Persistent check so we don't need these if/else statements everywhere

Reply:
the calling code could look like this
```
if options.Shorthand == "" {
getFlags(c.Command).IntVar(
```

Reply:
I've moved this code out of this PR, will do this in it's own PR.